addons/website/static/src/snippets/s_accordion/000.scss

```diff
@@ -0,0 +1,69 @@
+.s_accordion {
+
+    // Replace the default chevron with our Odoo UI icon to
+    // have more control on its color depending on the
+    // preset applied
+    .accordion-button:after {
+        margin: 0;
+        background: none;
+        font-family: 'odoo_ui_icons';
+        text-align: center;
+        content: '\e839';
+    }
+
+    .s_accordion_highlight {
+        .accordion-item {
+            border: 0;
+            border-radius: var(--#{$prefix}accordion-border-radius);
+            overflow: hidden;
+
+            // This removes the bg color when the item isn't displayed.
+            &:has(.collapsed) {
```

Comment:
I haven't found any other way to do it. 

Since this pseudo-class is more than 92% covered, I think we can use it.
https://caniuse.com/?search=%3Ahas()

Also, in this case, `has()` is defined for decorative purpose (it allows only the background of the active element to be displayed). This means that even in the worst scenario when the user's browser can't render this selector, he can still see the content without having any readability issue.


| rendering with`has()`✅ | rendering without `has()`❌ |
|--------|--------|
| <img width="1403" alt="Capture d’écran 2024-08-08 à 14 59 26" src="https://github.com/user-attachments/assets/6de50257-1e66-4fbe-99b7-5ee6c1d96795"> | <img width="1374" alt="Capture d’écran 2024-08-08 à 15 02 47" src="https://github.com/user-attachments/assets/432f658f-c7c7-4b8b-bb97-020940eb2a9a"> |

Reply:
Yes, I forgot to update my comment, but indeed it's not blocking for browsers that don't support it. :+1: That's good for me